lib/rollbar_transport.js

```diff
@@ -28,24 +28,27 @@ class RollbarTransport extends TransportStream {
   }
 
   log(info, callback) {
-    const {level, message} = info;
-    process.nextTick(() => {
```

Comment:
We're changing a lot of code where I guess it's unclear why it was there in the first place. The git history doesn't tell a good story, and without unit tests it's hard to say why that has been there in the first place, so I think it's ok.